src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -153,10 +155,9 @@ public async Task EndToEnd_NoAPI()
 
         newProjectDir.Create();
         privateNuGetAssets.Create();
-        var repoGlobalJson = Path.Combine("..", "..", "..", "..", "global.json");
-        File.Copy(repoGlobalJson, Path.Combine(newProjectDir.FullName, "global.json"));
```

Comment:
I think this was needed. We need to troubleshoot it.

Reply:
This global.json specifies SDK version. For SDK test framework, it only has the built SDK and its version changes over time. So it's not needed in SDK test. Does it make sense?